lib/internal/loader/Loader.js

```diff
@@ -1,15 +1,18 @@
 'use strict';
 
 const { getURLFromFilePath } = require('internal/url');
+const errors = require('internal/errors');
 
-const { createDynamicModule } = require('internal/loader/ModuleWrap');
-
-const ModuleMap = require('internal/loader/ModuleMap');
+const Registry = require('internal/loader/Registry');
 const ModuleJob = require('internal/loader/ModuleJob');
-const ModuleRequest = require('internal/loader/ModuleRequest');
-const errors = require('internal/errors');
+const defaultResolve = require('internal/loader/DefaultResolve');
+const createDynamicModule = require('internal/loader/CreateDynamicModule');
+const translators = require('internal/loader/Translators');
+
 const debug = require('util').debuglog('esm');
 
+const kDynamicTranslate = Symbol('dynamicTranslate');
```

Comment:
Why is this a symbol if it is essentially a publicly mutable property?

Reply:
I want to force people to use hook, in case we have any internal structure changes in the future, and I should probably do the same with resolve

Reply:
They can get this via `Object.getOwnPropertySymbols`, I don't think this is private.

Reply:
@bmeck i think the best way is to use an unexported weakmap but if you have a better idea lemme know